ipyleaflet/leaflet.py

```diff
@@ -2031,6 +2033,38 @@ def remove_legend_element(self, key):
         self.send_state()
 
 
+class ColormapControl(WidgetControl):
+    """ColormapControl class, with WidgetControl as parent class.
+
+    A control which contains a colormap.
+
+    Attributes
+    ----------
+    caption : str, default 'caption'
+        The caption of the colormap.
+    colormap: branca.colormap.ColorMap instance, default linear.YlOrRd_04
+        The colormap used for the effect.
+    value_min : float, default 0.0
+        The minimal value taken by the data to be represented by the colormap.
+    value_max : float, default 1.0
+        The maximal value taken by the data to be represented by the colormap.
+    """
+    caption = Unicode('caption')
+    colormap = Instance(LinearColormap, default_value=linear.YlOrRd_04)
```

Comment:
Do we require a linear color map, or just any color map?

Reply:
We can use Colormap Instance that is more general.